src/string.c

```diff
@@ -12,4 +12,39 @@ size_t strlen(char *string)
   }
 
   return len;
-}
\ No newline at end of file
+}
+
+i32 strcmp(char *str1, char *str2)
+{
+    i32 i=0;
+    while(str1[i] != '\0' && str2[i] != '\0' && (str1[i] == str2[i])) 
+    {
+        i++;
+    }
+    if(str1[i] == '\0' && str2[i] == '\0') 
+    {
+        return 0;
+    }
+    if(str1[i] == '\0' && str2[i] != '\0') 
+    {
+        return -1;
+    }
+    if(str1[i] != '\0' && str2[i] == '\0') 
+    {
+        return 1;
+    }
+}
+
+i32 strncmp(char *str1, char *str2, i32 n)
+{
+    i32 i=0;
```

Comment:
Please stick to the code style and indentation style of the rest of the project.